testsuite/features/step_definitions/common_steps.rb

```diff
@@ -870,6 +870,11 @@
   else # As Ubuntu has no support, must be CentOS/SLES_ES
     node.run('yum install wget', timeout: 600)
   end
+  registration_url = if $proxy.nil?
+                       "https://#{$server.full_hostname}/XMLRPC"
+                     else
+                       "https://#{$proxy.full_hostname}/XMLRPC"
+                     end
```

Comment:
I wonder if that's part of HTTP API tests, or that's also out of scope...

Reply:
I asked myself the same.

It's passed to `spacecmd` and this command line tool continues to use XML-RPC independently of my PR.